### PR TITLE
Highlight current page in the navigation bar

### PR DESCRIPTION
**Pull request by not-soham**
### Motivation:
As per,
- #502 

made changes to highlight current page in the navigation bar.
### Modifications:
- Updated `navigation.html`
- Updated active color to a variable `--color-active-nav-item`

### Result:
The current page will be highlighted in the navigation bar.

image
image

image
image

### CODE REVIEW

**Comment by not-soham** [comment]
@swift-ci please test

File: assets/stylesheets/_screen.scss [hunk 1]
```diff
@@ -219,11 +219,9 @@ cite {
     display: inline-block;
     margin-left: .5em;
 
-    &.active {
-      a {
-        font-weight: 700;
-        color: #F05138; // TODO: Use variable
-      }
+    .active a{
```
Comment (kaishin) [inline assets/stylesheets/_screen.scss:222]:
```suggestion
    .active a {
```

**Review by kaishin — APPROVED** [review]
Thanks! Just one minor nit.

**Comment by 0xTim** [comment]
@kaishin is this waiting on anything?

**Comment by 0xTim** [comment]
@swift-ci test

**Comment by 0xTim** [comment]
Ah it appears not, will merge

**Pull request merged**